docs/src/understanding-deis/components.md

```diff
@@ -41,18 +42,22 @@ The router component uses [Nginx][] to route traffic to application containers.
 
 ## Store
 
-The store component uses [Ceph][] to store data for Deis components
-which need to store state, including [Registry][], [Database][].
+The Store component uses [Minio][] to store data for Deis components
```

Comment:
this should probably say that the store component uses minio by default - you can configure the cluster to use other object stores